docs/testrunner-toolkit/configuration/playwright.md

```diff
@@ -4,6 +4,13 @@ title: "Configuration Syntax: Playwright"
 sidebar_label: Playwright
 ---
 
+export const Highlight = ({children, color}) => ( <span style={{
```

Comment:
Hm. Is it possible to publish that one at a higher level so that it's shared between the other configurations?

Reply:
Inline was the easy/lazy way of using MDX, but it looks like it is possible to abstract it via a component. 🤔 
https://v2.docusaurus.io/docs/markdown-features#embedding-react-components-with-mdx

Reply:
Updated :-)

Reply:
@FriggaHel your are the mvp 🥇